### PR TITLE
fixing getting undefiened for points when answering a question

### PR DESCRIPTION
**Pull request by ahmedqarmout2**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -570,8 +570,8 @@ app.post('/submitanswer', function(req, res) {
         req.session.user,
         req.body.answer,
         function(err, value) {
-            var result = value ? 'correct' : 'incorrect';
-            var status = value ? 200 : 500;
+            var result = value.correct ? value : 'incorrect';
```
Comment (esteeroramy) [inline server.js:573]: `var` -> `const`

**Review by esteeroramy — APPROVED** [review]

**Comment by esteeroramy** [comment]
minor change requested

File: server.js (hunk 1 above)
Comment (ahmedqarmout2) [reply server.js:573]: No difference really. If this was a huge function, then I would agree with you and change it right away. But its just a 2 lines function.

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Looks good to me, nice work.

**Review by Mohammed-JS-Khan — APPROVED** [review]
Looks good to me 👍

**Pull request merged**